moveit_configs_utils/moveit_configs_utils/launches.py

```diff
@@ -132,3 +133,56 @@ def generate_warehouse_launch(moveit_config):
     ld.add_action(reset_node)
 
     return ld
+
+
+def generate_move_group_launch(moveit_config):
+    ld = LaunchDescription()
+
+    ld.add_action(DeclareBooleanLaunchArg("debug", default_value=False))
+    ld.add_action(
+        DeclareBooleanLaunchArg("allow_trajectory_execution", default_value=True)
+    )
+    ld.add_action(
+        DeclareBooleanLaunchArg("publish_monitored_planning_scene", default_value=True)
+    )
+    # load non-default MoveGroup capabilities (space separated)
+    ld.add_action(DeclareLaunchArgument("capabilities", default_value=""))
+    # inhibit these default MoveGroup capabilities (space separated)
+    ld.add_action(DeclareLaunchArgument("disable_capabilities", default_value=""))
+
+    should_publish = LaunchConfiguration("publish_monitored_planning_scene")
+
+    move_group_configuration = {
+        "publish_robot_description_semantic": True,
+        "allow_trajectory_execution": LaunchConfiguration("allow_trajectory_execution"),
+        # Note: Wrapping the following values is necessary so that the parameter value can be the empty string
+        "capabilities": ParameterValue(
+            LaunchConfiguration("capabilities"), value_type=str
+        ),
+        "disable_capabilities": ParameterValue(
+            LaunchConfiguration("disable_capabilities"), value_type=str
+        ),
+        # Publish the planning scene of the physical robot so that rviz plugin can know actual robot
+        "publish_planning_scene": should_publish,
+        "publish_geometry_updates": should_publish,
+        "publish_state_updates": should_publish,
+        "publish_transforms_updates": should_publish,
+    }
+
+    move_group_params = [
+        move_group_configuration,
+        moveit_config.to_dict(),
+    ]
+
+    add_debuggable_node(
+        ld,
+        package="moveit_ros_move_group",
+        executable="move_group",
+        commands_file=str(moveit_config.package_path / "launch" / "gdb_settings.gdb"),
+        output="screen",
+        parameters=move_group_params,
+        extra_debug_args=["--debug"],
+        # Set the display variable, in case OpenGL code is used internally
+        additional_env={"DISPLAY": ":0"},
```

Comment:
is this really necessary? I don't think it's our responsibility to override DISPLAY

Reply:
Copied from [here](https://github.com/ros-planning/moveit2/blob/9e30bdc77d0a9386dfdc50766e782de8be155010/moveit_setup_assistant/templates/moveit_config_pkg_template/launch/move_group.launch#L85)